files/zh-cn/learn/javascript/building_blocks/functions/index.md

```diff
@@ -267,29 +323,29 @@ JavaScript 由于各种原因而建立，但主要是由于安全性和组织性
 
 ```js
 // first.js
-let name = "Chris";
+const name = "Chris";
 function greeting() {
-  alert("Hello " + name + ": welcome to our company.");
+  alert(`Hello ${name}: welcome to our company.`);
```

Comment:
If possible, without affecting the operation and the purpose of the example,  the strings and comments in the code can be translated - that will make the documentation more friendly. I notice there are some other examples.  Please forgive me for not pointing them out one by one.

```suggestion
  alert(`你好 ${name}：欢迎来到我们公司。`);
```